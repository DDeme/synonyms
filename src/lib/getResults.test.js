const getResults = require('./getResults')

describe('test of get results function', () => {
  const dictionary = {
    big: ['large'],
    large: ['huge', 'big'],
    huge: ['large'],
    small: ['little'],
    little: ['small'],
    apple: ['banana'],
    banana: ['apple'],
  }
  const queries = [
    'same same',
    'big huge',
    'huge big',
    'apple peach',
    'big tall',
    'peach PEACH',
  ]
  test('sample integration test for getResults', () => {
    expect(getResults(queries, dictionary)).toMatchSnapshot()
  })

  // TODO: cover more l
})
