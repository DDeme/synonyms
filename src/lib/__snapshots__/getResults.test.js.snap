// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test of get results function sample integration test for getResults 1`] = `
Array [
  "synonyms",
  "synonyms",
  "synonyms",
  "different",
  "different",
  "synonyms",
]
`;
